tools/maint/create_release.py

```diff
@@ -59,15 +59,15 @@ def main():
   branch_name = 'version_' + release_version
 
   # Create a new git branch
-  subprocess.check_call(['git', 'checkout', '-b', branch_name], cwd=root_dir)
+  subprocess.check_call(['git', 'checkout', '-b', branch_name, 'origin/main'], cwd=root_dir)
 
   # Create auto-generated changes to the new git branch
   subprocess.check_call(['git', 'add', '-u', '.'], cwd=root_dir)
   subprocess.check_call(['git', 'commit', '-m', f'Mark {release_version} as released'], cwd=root_dir)
+  print('New release created in branch: `%s`' % branch_name)
 
-  print('New relase created in branch: `%s`' % branch_name)
-
-  # TODO(sbc): Maybe create the tag too, and even push both to `origin`?
```

Comment:
is creating the tag something we want to do in this script, either in this PR or leave the TODO for that?

Reply:
Oh yeah, I was wasn't sure about that.  I leave the TODO for now.